spicy/src/rt/tests/parser.cc

```diff
@@ -217,6 +217,52 @@ TEST_CASE("waitForInput") {
     }
 }
 
+TEST_CASE("waitForInput with min") {
```

Comment:
What's the "with min" part in this test? Not seeing a different to the earlier test, but I may well be missing something.

Reply:
The difference is in the lambda which passes `min=3` while the previous test tries to read any input. The called function takes a different call path so I'd suggest we keep this case.